src/pages/Pool-V3/components/PoolList/index.tsx

```
@@ -192,27 +113,86 @@ const PoolList = ({ search }) => {
-  useEffect(() => {
-    if (Object.values(poolLiquidities).length > 0) {
-      sortDataSource({
-        [PoolColumnHeader.VOLUME]: SortType.DESC
-      } as any);
-    }
-  }, [poolLiquidities]);
+  const filteredPool = dataPool
+    .map((item) => {
+      let volumeV3 = 0;
+      if (item?.poolKey) {
+        const findPool = volumnePools && volumnePools.find((vo) => vo.poolAddress === item?.poolKey);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)